lms/config.py

```diff
@@ -104,6 +104,8 @@ def __post_init__(self):
     _Setting("mailchimp_digests_subaccount"),
     _Setting("mailchimp_digests_email"),
     _Setting("mailchimp_digests_name"),
+    # Point to the URL the web app runs on. Useful in celery tasks and the shell
+    _Setting("web_app_url"),
```

Comment:
I don't know if there's a nicer way of doing this.

I reckon that a nicer way that works if we eventually deployed this independently of the web worker is unlikely so a explicit envvar doesn't seem like a bad solution.